gimie/sources/github.py

```diff
@@ -117,15 +123,13 @@ def to_graph(self) -> Graph:
 
     def extract(self):
         """Extract metadata from target GitHub repository."""
-        if self._id is None:
-            self._id = self.path
-        self.name = urlparse(self.path).path.strip("/")
-        data = self._fetch_repo_data(self.path)
+        self.name = self.path
```

Comment:
What is the reason for calling this attribute `name` and using it instead of directly using `path`? I find it a bit confusing.

Reply:
right, we can simply use path